contracts/PeeranhaToken.sol

```diff
@@ -43,15 +42,17 @@ contract PeeranhaToken is ERC20Upgradeable, ERC20PausableUpgradeable, ERC20Cappe
    * - must be a period less then now.
   */
   function claimReward(uint16 period) external {
-    RewardLib.StatusReward storage rewardStatus = RewardLib.getRewardStatus(statusRewards, msg.sender, period);
```

Comment:
Why this was removed?
We still should use `rewardStatus` to track paid out rewards.
Remove `isPaid` from `UserRewards`.